package exeptionHandling;


public class assign1 {
	
	public static void main(String[] args) {	
		try {
			int arr1[]= new int[-2];
				
			
		}
		catch(Exception e)
		{
			System.out.println(e);
		}
	}

}
//multiple catch statements
//Exception are : 
